src/pymovements/gaze/gaze_dataframe.py

```diff
@@ -1478,14 +1541,14 @@ def _convert_time_units(self, time_unit: str | None) -> None:
                     pl.col('time').cast(pl.Int64),
                 )
 
-    def __str__(self: Any) -> str:
+    def __str__(self) -> str:
```

Comment:
you culd annotate with `typing.Self`, but probably not needed

Reply:
yeah, but as we don't typehint self in any other place in source, I would just prefer to be simple and consistent.